### PR TITLE
openshift-cli 4.13.17

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by cho-m — DISMISSED** [review]

File: Formula/o/openshift-cli.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class OpenshiftCli < Formula
   desc "OpenShift command-line interface tools"
   homepage "https://www.openshift.com/"
-  url "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.13.14/openshift-client-src.tar.gz"
+  url "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.13.15/openshift-client-src.tar.gz"
   sha256 "6b6ac6109dbbc8240a343d4a07661cf98ae008305bf432b58ca6b9c00c5a83d5"
```
Comment (cho-m) [inline Formula/o/openshift-cli.rb:5]: Dismissed review as looks like version number changed in URL but sha256 for tarball is the same.

Comment (iMichka) [reply Formula/o/openshift-cli.rb:5]: Looking at https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.13.15/sha256sum.txt. and https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.13.14/sha256sum.txt it did indeed not change. They maybe just re-release from the same source and made a rebuild of their binaries ...

Anyway  openshift-client-src-4.13.17-x86_64.tar.gz is out and this one has a different sha

**Comment by fxcoudert** [comment]
`The formula version (4.13.17) is newer than the version from `brew livecheck` (4.13.15).`

**Comment by chenrui333** [comment]
looks like indeed pre-release, closing for now.

image

**Comment by chenrui333** [comment]
#152399

**Pull request closed without merging**